src/shogun/regression/LeastAngleRegression.cpp

```diff
@@ -138,14 +143,15 @@ bool CLeastAngleRegression::train_machine(CFeatures* data)
 		for (int32_t j=0; j < n_fea; ++j)
```

Comment:
this transposition of the input matrix should be handled by eigen or linalg... doing this with a double nested loops is soo... 101 computer science